### PR TITLE
Change code formatter to black from yapf

### PR DESCRIPTION
**Pull request by RatanShreshtha**
Fixes https://github.com/python-trio/cookiecutter-trio/issues/16

### CODE REVIEW

File: {{cookiecutter.project_slug}}/CHEATSHEET.rst [hunk 1]
```diff
@@ -56,13 +56,13 @@ To run tests
 * Actually run the tests: ``pytest {{cookiecutter.package_name}}``
 
 
-To run yapf
+To run black
 -----------
```
Comment (njsmith) [inline {{cookiecutter.project_slug}}/CHEATSHEET.rst]: I think you might need to extend the underline to match the length of the previous line? ReST is weird.

File: {{cookiecutter.project_slug}}/CHEATSHEET.rst [hunk 2]
```diff
@@ -56,13 +56,13 @@ To run tests
...
 
-* Show what changes yapf wants to make: ``yapf -rpd setup.py
+* Show what changes black wants to make: ``black --diff setup.py
   {{cookiecutter.package_name}}``
 
-* Apply all changes directly to the source tree: ``yapf -rpi setup.py
+* Apply all changes directly to the source tree: ``black -rpi setup.py
```
Comment (njsmith) [inline {{cookiecutter.project_slug}}/CHEATSHEET.rst]: I don't think black takes `-rpi` options?

File: {{cookiecutter.project_slug}}/MANIFEST.in [hunk 3]
```diff
@@ -1,5 +1,5 @@
 include README.rst CHEATSHEET.rst LICENSE* CODE_OF_CONDUCT* CONTRIBUTING*
-include .coveragerc .style.yapf
```
Comment (njsmith) [inline {{cookiecutter.project_slug}}/MANIFEST.in]: I think you'll also want to delete this file from the repo?

**Comment by RatanShreshtha** [comment]
I have updated

File: {{cookiecutter.project_slug}}/ci/travis.sh [hunk 4]
```diff
@@ -56,19 +54,15 @@ fi
 pip install -U pip setuptools wheel
 
 if [ "$CHECK_FORMATTING" = "1" ]; then
-    pip install yapf==${YAPF_VERSION}
-    if ! yapf -rpd setup.py {{cookiecutter.package_name}}; then
+    pip install black
+    if ! black --diff setup.py {{cookiecutter.package_name}}; then
```
Comment (njsmith) [inline {{cookiecutter.project_slug}}/ci/travis.sh]: Does this work? I think you might need to add `--check` as well, or else `black` always reports success, even if it needed to make changes.

Comment (RatanShreshtha) [reply {{cookiecutter.project_slug}}/ci/travis.sh]: yeah, I am updating it

**Comment by njsmith** [comment]
Thanks!

**Pull request merged**